impl/src/main/scala/io/guanaco/alerta/impl/AlertaRoutes.scala

```diff
@@ -20,6 +20,7 @@ class AlertaRoutes(val apiUrl: String, val environment: String) extends RouteBui
     //format: OFF
     from(getEndpoint(Alerta.ALERT_QUEUE_NAME))
       .transform(method(Helper()))
+      .bean(Authentication())
       .setHeader(CONTENT_TYPE, constant("application/json"))
```

Comment:
Hier zitten nu in de Camel route 3 stappen om de HTTP request klaar te maken, met 2 verschillende helper classes en dan nog een `setHeader` DSL call.

Misschien kunnen we dit nog refactoren naar één helper class & method (bv. met de naam `prepareHttpRequest`